db/migrate/20230329183634_add_posts_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostsRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :posts, null: false, foreign_key: true
```

Comment:
- Hi :clap: , in my previous review comment, the same solution is applicable here. it should be
`add_reference :likes, :post, null: false, foreign_key: true`. let's kindly fix this Please :pray: